src/pages/SignUp/Index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, {FC, useState} from 'react';
+import {SignUpScreenProps} from '../../utils/LoginUserNavigation';
+import {WelcomeText} from '../../components/molecules/WelcomView';
+import Back from '../../assets/images/BackIcon2.svg';
+import {Formik, Field} from 'formik';
+import {
+  Container,
+  MyAwareScrollView,
+  MySafeAreaView,
+} from '../Login/Styles';
+import {BackImageView, WelcomeTextSignupView, SignupInputView, ButtonSignupView} from './Styles';
+import {ValidationSchemaSignUp} from '../../utils/validation/Validation';
+import {InputWithImage} from '../../components/atoms/Inputs/Index';
+import {Buttons} from '../../components/atoms/Buttons/Index';
+import PrivacyPolicy from '../../components/molecules/PrivacyPolicyView';
+import String from '../../assets/strings/Strings';
+import {TouchableWithoutFeedback} from 'react-native';
+
+interface CheckProps {
+  checkBtn: boolean;
+}
+interface SecureTextProps {
+  secure?: boolean;
+  secureConfirmPassword?: boolean;
+}
+const SignUp: FC<SignUpScreenProps> = props => {
+  const [check, setCheck] = useState<CheckProps>({checkBtn: false});
+  const [secureText, setSecureText] = useState<SecureTextProps>({
+    secure: true,
+    secureConfirmPassword: true,
+  });
+
+  const handleCheck = () => {
+    setCheck({checkBtn: !check.checkBtn});
+  };
+
+  const handleSecurePassword = () => {
+    setSecureText({
+      secure: !secureText.secure,
+      secureConfirmPassword: secureText.secureConfirmPassword,
+    });
+  };
+
+  const handleSecureConfirmPassword = () => {
+    setSecureText({
+      secureConfirmPassword: !secureText.secureConfirmPassword,
+      secure: secureText.secure,
+    });
+  };
+
+  return (
+    <MySafeAreaView>
+      <Container>
+        <MyAwareScrollView>
+          <TouchableWithoutFeedback
+            onPress={() => props.navigation.push('Login')}>
```

Comment:
no using inline functions... create a separate function here and use useCallback